Automattic/wp-calypso
# Staging sites: Disable paid themes

Related to https://github.com/Automattic/dotcom-forge/issues/2042

## Proposed Changes

This pull request effectively disables the purchase functionality for staging sites. As paid themes cannot be acquired for a staging site, given the absence of a corresponding plan, this PR serves to guarantee that such purchases are not possible.

## Testing Instructions



### Test 

1. Create a new atomic site.
2. Navigate to `/themes/<your site>`
3. For a paid theme (Example can be found in the pictures below)
4. Ensure you don't see any buttons that allow you to purchase a theme (like shown below).
5. Ensure that by clicking the notice banner you are navigated to the theme page of the production site.

**NOTE**: The copy and the icon of the banner may be subject to change. I'm waiting for @javierarce for the final copy / icon of the banner.

| **Before** | **After** |
| :---:         |    :---:       |
|![paid_staging_before](https://user-images.githubusercontent.com/497103/229774904-3feca3b2-eb89-43ed-8f0d-00551ace3678.png)|![paid_staging_after](https://user-images.githubusercontent.com/497103/229774974-522c1e26-87eb-413e-88c7-c2934a171cde.png)|



## Pre-merge Checklist




**CLOSING AS I RENAMED THIS BRANCH**: https://github.com/Automattic/wp-calypso/pull/75274
